### PR TITLE
Artifact rename

### PR DESCRIPTION
**Pull request by seglo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,48 @@
 Alpakka Kafka [![scaladex-badge][]][scaladex] [![maven-central-badge][]][maven-central] [![gh-actions-badge][]][gh-actions]
```
Comment (nvollmar) [inline README.md:1]: Missed one

**Review by nvollmar — APPROVED** [review]
lgtm

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,33 +1,27 @@
-# Welcome! Thank you for contributing to Alpakka Kafka!
+# Welcome! Thank you for contributing to the Pekko Kafka connector!
```
Comment (pjfanning) [inline CONTRIBUTING.md:1]: Minor nit: could the first reference to Pekko be `Apache Pekko`? It's a bit repetitive to continuously use the full form but I think it's better to start with it.

File: project/project-info.conf [hunk 3]
```diff
@@ -57,7 +57,7 @@ project-info {
       }
     ]
   }
-  clusterSharding: ${project-info.shared-info} {
+  cluster-sharding: ${project-info.shared-info} {
     title: "Alpakka Kafka Cluster Sharding"
```
Comment (pjfanning) [inline project/project-info.conf:61]: is it too early to update the title to `Apache Pekko Connectors Kafka Cluster Sharding` or some such?

Comment (seglo) [reply project/project-info.conf:61]: I was thinking of taking care of that with the docs updates.

File: CONTRIBUTING.md (hunk 2 above)
Comment (seglo) [reply CONTRIBUTING.md]: Yes. Updated.

**Pull request merged**